### PR TITLE
Removed unnecessary cast to float in Gradients.java and Point.java.

### LINKED ISSUE #79: [Source Code]: Remove unnecessary cast to float.
### What's Up?

From the SonarCloud quality tests, it seems there are some unnecessary casts to float.

### Code of Conduct

- [X] I agree to follow FastJ's Code of Conduct

### PR DESCRIPTION
**Pull request by Sammie156**
# Removing Unnecessary Float Casts
Resolves #79.

## Other Changes
Changed float casting in some places in both Gradients.java and Point.java

### CODE REVIEW

**Comment by Sammie156** [comment]
POOOOOGGG!! ALL PASSED!!

File: src/main/java/tech/fastj/math/Point.java [hunk 1]
```diff
@@ -486,8 +486,8 @@ public Point rotate(int angle) {
         float sineOfAngle = (float) Math.sin(angleInRadians);
         float cosineOfAngle = (float) Math.cos(angleInRadians);
 
-        float rotatedX = ((float) x * cosineOfAngle) + ((float) y * sineOfAngle);
-        float rotatedY = ((float) -x * sineOfAngle) + ((float) y * cosineOfAngle);
+        float rotatedX = (x * cosineOfAngle) +  (y * sineOfAngle);
```
Comment (lucasstarsz) [inline src/main/java/tech/fastj/math/Point.java:489]: Looks like you added an extra space there by accident. Mind removing it?

**Review by lucasstarsz — COMMENTED** [review]
Thanks for contributing to FastJ! We really appreciate it 👍🏾 
Please take some time to review the changes needed before we can merge your PR.

File: src/main/java/tech/fastj/math/Point.java (hunk 1 above)
Comment (Sammie156) [reply src/main/java/tech/fastj/math/Point.java:489]: Sure Man!

**Review by lucasstarsz — APPROVED** [review]
Looks good to me 👍🏾

**Comment by Sammie156** [comment]
WoohOOO!!

**Comment by lucasstarsz** [comment]
Thanks again for contributing to FastJ! We hope to see you again sometime soon ❤️

**Pull request merged**